chainstate/src/rpc/mod.rs

```diff
@@ -142,6 +142,12 @@ trait ChainstateRpc {
     #[method(name = "staker_balance")]
     async fn staker_balance(&self, pool_address: String) -> RpcResult<Option<Amount>>;
 
+    /// Returns the pool data of the pool associated with the given pool address.
+    ///
+    /// Returns `None` (null) if the pool is not found.
+    #[method(name = "pool_data")]
+    async fn pool_data(&self, pool_address: String) -> RpcResult<Option<RpcPoolData>>;
```

Comment:
It's somewhat inconsistent that we have separate functions for various pool-related info, such as `staker_balance`, and then also this `pool_data`. I'd probably add specific functions that you need (`pool_decommission_destination`?) instead of this common one.

Reply:
Replaced with get pool decommission destination